libs/sdk-bindings/makefile

```diff
@@ -14,21 +14,21 @@ init:
 	cargo install cargo-ndk
 
 ## all: Compile iOS, Android
-all: swift-ios swift-darwin kotlin
+all: swift-ios swift-darwin bindings-swift kotlin
 
 ios-universal: $(SOURCES)		
 	mkdir -p ../target/ios-universal/release
 	cargo build --release --target aarch64-apple-ios ;\
 	cargo build --release --target x86_64-apple-ios ;\
-	lipo -create -output ../target/ios-universal/release/libbreez_sdk_bindings.a ../target/aarch64-apple-ios/release/libbreez_sdk_bindings.a ../target/x86_64-apple-ios/release/libbreez_sdk_bindings.a	
-
+	cargo build --release --target aarch64-apple-ios-sim ;\
+	lipo -create -output ../target/ios-universal/release/libbreez_sdk_bindings.a ../target/aarch64-apple-ios-sim/release/libbreez_sdk_bindings.a ../target/x86_64-apple-ios/release/libbreez_sdk_bindings.a
 
 darwin-universal: $(SOURCES)
 	mkdir -p ../target/darwin-universal/release
-	cargo lipo --release --targets aarch64-apple-darwin		
-	cargo lipo --release --targets x86_64-apple-darwin		
```

Comment:
Could we switch this to using `cargo build` directly, like we do for the iOS targets? 🤔

Reply:
I think you are probably right about it.